viewing_party/party.py

```diff
@@ -1,23 +1,147 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
 
+    if title and genre and rating:
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie

```

Comment:
When we see something like this, returning the hard coded dictionary is going to be a more conventional approach:

```suggestion
        return {
             "title": title,
             "genre": genre,
             "rating": rating
        }
```